exercises/parallel-letter-frequency/parallel_letter_frequency_test.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+from collections import Counter
+import unittest
+
+from parallel_letter_frequency import calculate
+
```

Comment:
Could you please also leave a comment stating what version of `canonical-data.json` the tests were adopted as discussed in #784 if aplicable?
The format is:
```text
# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
```

Reply:
I was unable to find a canonical-data.json in https://github.com/exercism/problem-specifications/tree/master/exercises/parallel-letter-frequency. I noticed that there is an open task for creating one (https://github.com/exercism/problem-specifications/issues/574). Should I still add the comment?

Reply:
@forgeRW probably. You can point out that there was no `canonical-data.json` at the time